src/typeahead/typeahead.js

```diff
@@ -343,6 +344,7 @@ var Typeahead = (function() {
 
     close: function close() {
       if (this.isOpen() && !this.eventBus.before('close')) {
+        this.input.$input.attr("aria-expanded", false);
```

Comment:
I have noticed your tests are broken because $input is not available in this context, if possible create the function bellow at `input.js` and call it here to set `aria-expanded`. This will fix all the broken tests.

``` 
setAriaExpanded: function setAriaExpanded(value) {
    this.$input.attr('aria-expanded', value);
},
```

Reply:
Thanks!